src/pages/patientView/PatientViewPage.tsx

```diff
@@ -544,6 +552,72 @@ export class PatientViewPageInner extends React.Component<
         return this.patientViewPageStore;
     }
 
+    @action.bound
+    private handleNavigationControlClick() {
+        const patientIdentifiers = this.pageStore.patientIdsInCohort.map(p => {
+            const patientIdParts = p.split(':');
+            return {
+                patientId: patientIdParts[1],
+                studyId: patientIdParts[0],
+            };
+        });
+        const queriedStudies: Set<string> = new Set(
+            patientIdentifiers.map(p => p.studyId)
+        );
+
+        const studyPage = window.open(
+            buildCBioPortalPageUrl(`study`, {
+                id: Array.from(queriedStudies).join(','),
+            }),
+            '_blank'
+        );
+        if (patientIdentifiers.length > 0) {
+            (studyPage as any).studyPageFilter = `filterJson=${JSON.stringify({
+                patientIdentifiers,
+            })}`;
+        }
+    }
+
+    @action.bound
+    private handleDeletePatient(deleteAtIndex: number) {
```

Comment:
in my testing, this doesn't work on a big cohort (e.g. >200) when the items are removed from the hash.  we end up stuck on a single patient.  not totally sure why that would be.